modules/google-maps/src/utils.js

```diff
@@ -204,9 +204,8 @@ export function getViewPropsFromCoordinateTransformer(map, transformer) {
   const focalDistance = 0.5 * projectionMatrix[5];
 
   return {
-    // Using external gl context - do not set css size
-    width: false,
-    height: false,
```

Comment:
Why remove this? It prevents Deck from scaling the gl context size to device pixel ratio. Removing ii=t will break context sharing.